include/openthread/border_routing.h

```diff
@@ -133,6 +133,8 @@ typedef enum
  * Initializes the Border Routing Manager on given infrastructure interface.
  *
  * @note  This method MUST be called before any other otBorderRouting* APIs.
+ * @note  This method can be re-called to change the infrastructure interface, but the Border Routing Manager should be
+ *        disabled first, and re-enabled after.
```

Comment:
Change the line below about `OT_ERROR_INVALID_STATE` error (now it says, "The Border Routing Manager has already been initialized")?

Maybe: "The Border Routing Manager is in a state other than disabled or uninitialized"?